.gitignore

```diff
@@ -3,6 +3,10 @@
 # C extensions
 *.so
 
+# vim
+*~
+.*.swp
```

Comment:
I don't object to these but for your own convenience it might be better to add them to your global gitignore file :)